catalyst-gateway/bin/src/settings/mod.rs

```diff
@@ -128,6 +144,22 @@ struct EnvVars {
     /// The Address of the Event DB.
     event_db_password: Option<StringEnvVar>,
 
+    /// Sets the maximum number of connections managed by the pool.
+    /// Defaults to 10.
+    event_db_max_size: u32,
```

Comment:
Lets make a separate structure for all these event-db settings, the same as we have for the chain follower and signed docs.

Reply:
added event db settings